file_storage/admin.py

```diff
@@ -1,3 +1,11 @@
 from django.contrib import admin
+from .models import *
```

Comment:
Actually, it's best-practice to indicate that this is a relative import, to avoid collisions with the universe of possible module names. I think it might be required in py3 (relative import turned off by default?) The _real_ problem with this line is the * :-) This pollutes the namespace, so you can get accidental overrides